utils/sl/prereq_tools/base.py

```diff
@@ -284,7 +287,7 @@ def run_commands(self, commands, subdir=None):
                 retval = True
             else:
                 print('RUN: %s' % command)
-                if subprocess.call(command, shell=True,
+                if subprocess.call(command, shell=True,       # nosec
```

Comment:
This probably wants looking at properly as using shell=True is never a good idea.

Reply:
Have done tests on this, split 'command' to a list and remove the shell=True.
However, it causing build failed. I agree we need to do research and more test on this.